inc/translations.php

```diff
@@ -75,38 +69,58 @@ private function get_locale_data() {
 		 *
 		 * @since 2.3.5
 		 *
-		 * @param $locales
+		 * @param array $locales All locale info for AIOSEOP from translate.wordpress.org.
```

Comment:
Nice to see documentation added. It's just the incorrect format with WP Documentation Standards.

`var` is also intended for class variables. For array params it is `type`

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-1-parameters-that-are-arrays